module/zfs/zfs_znode.c

```diff
@@ -1387,11 +1387,20 @@ zfs_zinactive(znode_t *zp)
 	 * closed.  The file will remain in the unlinked set.
 	 */
 	if (zp->z_unlinked) {
+		boolean_t toobig = toobig = zp->z_size > zp->z_blksz * zfs_delete_blocks;
```

Comment:
typo here, assigning `toobig` to itself?

Reply:
Oops, fixed.